src/main/java/Duke.java

```diff
@@ -1,10 +1,40 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    static ArrayList<Task> list = new ArrayList<Task>();
+
+    public static void printTasks() {
+        System.out.println("Here are the tasks in your list:");
+        for (Task task : list) {
+            System.out.println(String.format("%d. [%c] %s", task.getTaskId(),
+                    task.isDone() ? '✓' : '✗', task.getTaskName()));
+        }
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+        String logo = " ____         _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+        Scanner scan = new Scanner(System.in);
+        String userInput = scan.nextLine();
+        while (!userInput.equals("bye")) {
```

Comment:
Might want to apply the coding standard of "making the happy path prominent". 

But no big deal. Cheers bro :)

https://nus-cs2113-ay1920s2.github.io/website/se-book-adapted/chapters/codeQuality.html#code-quality